testSrc/unit/io/flutter/run/daemon/DaemonEventTest.java

```diff
@@ -158,6 +163,12 @@ public void canRecieveAppStopped() {
     checkLog("AppStopped: 42");
   }
 
+  @Test
+  public void canRecieveAppStoppedWithError() {
+    send("app.stop", curly("appId:42", "error:foobar"));
```

Comment:
I'm not sure why this works when foobar isn't quoted. The curly method doesn't automatically quote the value. Perhaps GSON is handling it?

But if it passes I guess it's okay.